feat(content): update plugins (#157)
glpi-project/doc: source/modules/configuration/plugins.rst

When you uninstall a plugin using the uninstall button in the plugin list or in the marketplace UI, it does not remove the plugin code and the plugin will still show in the list of plugins to re-install.

To permanently remove the plugin, it is necessary to delete the plugin's folder.
After removing the plugin's folder there should be a new "cleanup" action available for the plugin in GLPI. This will remove the reference to the plugin from the database.

.. image:: images/uninstall_plugin.png
   :alt: Uninstall plugin
   :scale: 100%

Plugins and Update GLPI
-----------------------

> When GLPI is updated, you can suspend the plugins.

Reviewer: ```suggestion
When GLPI is updated, the plugins execution is suspended.
```

> This allows you to preserve the state of a plugin before the GLPI update and restore it to its previous state once the update is complete.

Reviewer: ```suggestion
This allows you to preserve the state of a plugin before the GLPI update and restore it to its previous state once the update is complete.
The plugins execution will be automatically resumed when the update corresponds to a bugfix update only (e.g. 11.0.x to 11.0.y), but this will have to be done manually in case of a major or intermediate update (11.0.x to 11.1.y or 12.0.z).
```
